src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -326,12 +348,17 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
     isLoading: isEstimatedPoolAssetFeesDataLoading,
     isError: isEstimatedPoolAssetFeesDataError,
   } = useQuoteEstimatedFeesQuery({
+    // Sym opportunities do *not* require a pool asset Tx, all we need is a RUNE Tx to trigger the withdraw
```

Comment:
do we know or have we tried asym withdrawal just from the rune side? obviously there is logic related to asset/rune address on the position, so wondering if that would be a cheaper way to withdraw asym if wallet supports rune?

Reply:
disregard me. I am seeing it is attached to the from address